kubernetes_deploy/production/deployment.yml

```diff
@@ -15,8 +16,16 @@ spec:
           httpGet:
             path: /ping.json
             port: 80
-          initialDelaySeconds: 3
-          periodSeconds: 3
+          initialDelaySeconds: 5
+          timeoutSeconds: 1
+          periodSeconds: 10
```

Comment:
I'm curious, could you expand on why these timeout and period durations were chosen, please? (I found the initial delay mentioned in the description.)

Reply:
These are the Kubernetes defaults made explicit. See https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#configure-probes